web/packages/api/src/environment.ts

```diff
@@ -532,4 +532,138 @@ export const SNOWBRIDGE_ENV: { [id: string]: SnowbridgeEnvironment } = {
             GRAPHQL_API_URL: "https://data.snowbridge.network/graphql",
         },
     },
+    westend_sepolia: {
+        name: "westend_sepolia",
+        ethChainId: 11155111,
+        locations: [
+            {
+                id: "ethereum",
+                name: "Ethereum",
+                type: "ethereum",
+                destinationIds: ["assethub", "muse"],
+                erc20tokensReceivable: [
+                    {
+                        id: "WETH",
+                        address: "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
+                        minimumTransferAmount: 1_000_000_000_000n,
+                    },
+                    {
+                        id: "vETH",
+                        address: "0xc3d088842dcf02c13699f936bb83dfbbc6f721ab",
+                        minimumTransferAmount: 1n,
+                    },
+                    {
+                        id: "MUSE",
+                        address: "0xb34a6924a02100ba6ef12af1c798285e8f7a16ee",
+                        minimumTransferAmount: 1n,
+                    },
+                ],
+            },
+            {
+                id: "assethub",
+                name: "Asset Hub",
+                type: "substrate",
+                destinationIds: ["ethereum"],
+                paraInfo: {
+                    paraId: 1000,
+                    destinationFeeDOT: 0n,
+                    skipExistentialDepositCheck: false,
+                    addressType: "32byte",
+                    decimals: 12,
+                    maxConsumers: 16,
+                },
+                erc20tokensReceivable: [
+                    {
+                        id: "WETH",
+                        address: "0xfff9976782d46cc05630d1f6ebab18b2324d6b14",
+                        minimumTransferAmount: 15_000_000_000_000n,
+                    },
+                    {
+                        id: "vETH",
```

Comment:
Lets remove vETH until its registered. This case was for bifrost in rococo. They are not in Westend so I do not think we need this.